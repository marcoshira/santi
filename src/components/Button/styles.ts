import styled, { css } from 'styled-components';

export const Wrapper = styled.button`
  ${({ theme }) => css`
    padding: ${theme.spacings.xxsmall} ${theme.spacings.xsmall};
    background: inherit;
    color: ${theme.colors.teal};
    /* border: 0.5px solid ${theme.colors.teal}; */
    border: none;
    font-size: ${theme.font.sizes.ltmedium};
    font-weight: 500;
    margin-left: ${theme.spacings.xxsmall};
    margin-top: 5px;
    margin-bottom: 4px;
    position: relative;
    transition: all 300ms ease-in-out;

    &:disabled {
      background: ${theme.colors.mediumGrey};
      color: ${theme.colors.white};
      border: 2px solid ${theme.colors.mediumGrey};
      transition: none;

      &:hover {
        background: ${theme.colors.mediumGrey};
        color: ${theme.colors.white};
        border: 2px solid ${theme.colors.mediumGrey};
        cursor: not-allowed;
      }

      &::after {
        content: none;
      }
    }
    &:hover {
      color: ${theme.colors.thirdColor};
      cursor: pointer;
    }

    &::after {
      content: '';
      position: absolute;
      bottom: 0.2rem;
      left: 50%;
      width: 0;
      height: 0.2rem;
      background: ${theme.colors.thirdColor};
      transition: all 200ms ease-in-out;
    }

    &:hover::after {
      left: 25%;
      width: 50%;
    }
  `}
`;
