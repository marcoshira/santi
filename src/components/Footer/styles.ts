import styled, { css } from 'styled-components';

export const Container = styled.footer`
  ${({ theme }) => css`
    text-align: center;
    border-top: 0.2rem solid ${theme.colors.mediumGrey};

    a {
      color: inherit;
      text-decoration: none;
    }
  `}
`;
