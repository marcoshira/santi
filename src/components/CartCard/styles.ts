import styled, { css } from 'styled-components';
import { Wrapper as Btn } from '../Button/styles';
import { Title } from '../Heading/styles';

export const Wrapper = styled.div`
  ${({ theme }) => css`
    display: flex;
    flex-direction: row;
    width: 100%;
    overflow-x: hidden;
    box-shadow: 3px 3px 15px ${theme.colors.teal};
    border-radius: 15px;
    color: ${theme.colors.navy};
    background: ${theme.colors.footerWhite};
    position: relative;
    padding: ${theme.spacings.small} 0;
    margin-bottom: ${theme.spacings.ltmedium};

    @media ${theme.media.lteMedium} {
      flex-direction: column;
      min-height: 50rem;
      width: 85%;
      justify-content: center;
      align-items: center;
      margin: ${theme.spacings.medium} auto;
    }

    a {
      cursor: pointer;
      &:hover {
        img {
          opacity: 0.6;
        }
      }
    }

    img {
      border-radius: 10px;
      height: 20rem;
      transition: ${theme.transitions.faster};

      @media ${theme.media.lteMedium} {
        height: 15rem;
      }
    }

    ${Title} {
      margin: 0;
    }
  `}
`;

export const ContentWrapper = styled.div`
  ${({ theme }) => css`
    width: 55%;
    a {
      text-decoration: none;
      color: ${theme.colors.navy};
      transition: ${theme.transitions.faster};
      &:hover {
        color: ${theme.colors.thirdColor};
      }
    }

    @media ${theme.media.lteMedium} {
      width: 100%;
      padding: ${theme.spacings.medium};
    }
  `}
`;

export const ImageWrapper = styled.div`
  display: flex;
  justify-content: center;
  width: 45%;
`;

export const ButtonContainer = styled.div`
  ${({ theme }) => css`
    display: flex;
    justify-content: space-between;
    flex-direction: row;
    margin-top: 2rem;
    margin-right: 2rem;

    ${Btn} {
      min-width: 48%;

      margin: 0rem;
      margin-right: 1rem;
      border: 2px solid ${theme.colors.teal};
      background: ${theme.colors.footerWhite};
      color: ${theme.colors.teal};

      &:hover {
        border: 2px solid ${theme.colors.thirdColor};
        background: ${theme.colors.footerWhite};
        color: ${theme.colors.thirdColor};
      }

      &::after {
        content: none;
      }

      :nth-of-type(2) {
        background: ${theme.colors.teal};
        color: ${theme.colors.footerWhite};

        &:hover {
          border: 2px solid ${theme.colors.teal};
          background: ${theme.colors.hoverTeal};
        }
      }
    }

    @media ${theme.media.lteMedium} {
      flex-direction: column;
      margin-right: 0rem;

      ${Btn} {
        width: 100%;
        margin: ${theme.spacings.tiny} 0;
      }
    }
  `}
`;
