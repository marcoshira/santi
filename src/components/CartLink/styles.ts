import styled, { css } from 'styled-components';
import { Title } from '../Heading/styles';

export const Wrapper = styled.div`
  ${({ theme }) => css`
    position: absolute;
    left: 40rem;
    top: 9rem;
    width: calc(100vw - 55rem);

    @media ${theme.media.lteMedium} {
      width: 100%;
      left: auto;
      top: 6rem;
      margin: 0 auto;

      > ${Title} {
        margin-left: ${theme.spacings.ltlarge};
      }
    }
  `}
`;
