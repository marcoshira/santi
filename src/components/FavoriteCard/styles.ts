import styled, { css } from 'styled-components';
import { Wrapper as Btn } from '../Button/styles';
import { Title } from '../Heading/styles';

export const Wrapper = styled.div`
  ${({ theme }) => css`
    display: flex;
    flex-direction: row;
    width: 100%;
    box-shadow: 3px 3px 15px ${theme.colors.teal};
    border-radius: 15px;
    color: ${theme.colors.navy};
    background: ${theme.colors.footerWhite};
    position: relative;
    padding: ${theme.spacings.small} 0;
    margin-bottom: ${theme.spacings.ltmedium};

    @media ${theme.media.lteMedium} {
      flex-direction: column;
      min-height: 53rem;
      width: 85%;
      justify-content: center;
      align-items: center;
      margin: ${theme.spacings.medium} auto;
      padding-bottom: 0;
    }

    a {
      cursor: pointer;
      &:hover {
        img {
          opacity: 0.6;
        }
      }
    }

    img {
      border-radius: 10px;
      height: 20rem;
      transition: ${theme.transitions.faster};

      @media ${theme.media.lteMedium} {
        height: 15rem;
      }
    }

    ${Title} {
      margin: 0;
    }
  `}
`;

export const ContentWrapper = styled.div`
  ${({ theme }) => css`
    width: 55%;
    a {
      text-decoration: none;
      color: ${theme.colors.navy};
      transition: ${theme.transitions.faster};
      &:hover {
        color: ${theme.colors.thirdColor};
      }
    }
    svg {
      height: 3rem;
      position: absolute;
      top: 2rem;
      right: 2rem;
      color: ${theme.colors.teal};
      cursor: pointer;
      transition: ${theme.transitions.fast};

      &:hover {
        color: ${theme.colors.hoverTeal};
      }
    }

    @media ${theme.media.lteMedium} {
      width: 100%;
      padding: ${theme.spacings.medium};
    }
  `}
`;

export const ImageWrapper = styled.div`
  ${({ theme }) => css`
    width: 45%;
    display: flex;
    justify-content: center;

    img {
    }
  `}
`;

export const ButtonContainer = styled.div`
  ${({ theme }) => css`
    display: flex;
    justify-content: space-around;
    flex-direction: row;
    margin-top: 3rem;
    ${Btn} {
      min-width: 47%;

      margin: 0rem;
      margin-right: 1rem;
      border: 2px solid ${theme.colors.teal};
      background: ${theme.colors.footerWhite};
      color: ${theme.colors.teal};

      &:hover {
        border: 2px solid ${theme.colors.thirdColor};
        background: ${theme.colors.footerWhite};
        color: ${theme.colors.thirdColor};
      }

      &::after {
        content: none;
      }

      :nth-of-type(2) {
        background: ${theme.colors.teal};
        color: ${theme.colors.footerWhite};

        &:hover {
          border: 2px solid ${theme.colors.teal};
          background: ${theme.colors.hoverTeal};
        }
      }
    }

    @media ${theme.media.lteMedium} {
      flex-direction: column;
      margin-right: 0rem;

      ${Btn} {
        width: 100%;
        margin: ${theme.spacings.tiny} 0;
      }
    }
  `}
`;

export const InputWrapper = styled.div`
  ${({ theme }) => css`
    display: flex;
    width: 50%;
    justify-content: left;
    flex-flow: row nowrap;
    font-size: 1.6rem;
    margin-top: 3rem;

    @media ${theme.media.lteMedium} {
      width: 100%;
      margin-top: 1rem;
    }

    label {
      margin: 8px ${theme.spacings.ltlarge} 0 0;
      color: ${theme.colors.darkGrey};

      @media ${theme.media.lteMedium} {
        margin: 7px ${theme.spacings.hero} 0 0;
      }
    }

    input {
      width: 40px;
      display: flex;
      text-align: center;
      align-items: center;
      justify-content: center;
      border: 1px solid ${theme.colors.grey};
      height: 35px;
    }
  `}
`;

export const Button = styled.button`
  ${({ theme }) => css`
    padding: 0 10px;
    background: ${theme.colors.footerWhite};
    border: 1px solid ${theme.colors.grey};
  `}
`;
