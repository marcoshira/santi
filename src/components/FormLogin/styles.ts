import styled, { css } from 'styled-components';
import { Wrapper as Button } from '../Button/styles';

export const Wrapper = styled.form``;

export const ButtonWrapper = styled.div`
  ${({ theme }) => css`
    margin-top: ${theme.spacings.medium};
    display: flex;
    flex-direction: row;
    justify-content: space-between;

    ${Button} {
      a {
        text-decoration: none;
        color: ${theme.colors.footerWhite};
      }
      width: 15rem;
      border: 2px solid ${theme.colors.teal};
      background: ${theme.colors.teal};
      color: ${theme.colors.footerWhite};
      border-radius: ${theme.spacings.tiny};
      margin-left: 0;
      &:hover {
        background: ${theme.colors.hoverTeal};
      }
      &::after {
        height: 0;
      }
      &:disabled {
        background: ${theme.colors.mediumGrey};
        color: ${theme.colors.footerWhite};
        border: 2px solid ${theme.colors.mediumGrey};

        &:hover {
          background: ${theme.colors.mediumGrey};
          color: ${theme.colors.white};
          border: 2px solid ${theme.colors.mediumGrey};
          cursor: not-allowed;
        }

        &::after {
          content: none;
        }
      }

      @media ${theme.media.lteSmall} {
        width: 100%;
        padding: ${theme.spacings.xsmall} ${theme.spacings.xsmall};
      }
    }
  `}
`;
