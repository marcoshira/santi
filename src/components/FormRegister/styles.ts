import styled, { css } from 'styled-components';
import { Wrapper as Button } from '../Button/styles';

export const Wrapper = styled.form`
  ${({ theme }) => css``}
`;

export const ButtonWrapper = styled.div`
  ${({ theme }) => css`
    margin-top: ${theme.spacings.medium};
    display: flex;
    flex-direction: row;
    justify-content: space-between;

    ${Button} {
      a {
        text-decoration: none;
        color: ${theme.colors.footerWhite};
      }
      width: 15rem;
      border: 2px solid ${theme.colors.teal};
      background: ${theme.colors.teal};
      color: ${theme.colors.footerWhite};
      border-radius: ${theme.spacings.tiny};
      margin-left: 0;
      &:hover {
        background: ${theme.colors.hoverTeal};
      }
      &::after {
        height: 0;
      }
      &:disabled {
        background: ${theme.colors.mediumGrey};
        color: ${theme.colors.footerWhite};
        border: 2px solid ${theme.colors.mediumGrey};

        &:hover {
          background: ${theme.colors.mediumGrey};
          color: ${theme.colors.white};
          border: 2px solid ${theme.colors.mediumGrey};
          cursor: not-allowed;
        }

        &::after {
          content: none;
        }
      }
    }
  `}
`;

export const ErrorMessage = styled.p`
  ${({ theme }) => css`
    color: ${theme.colors.thirdColor};
    font-size: ${theme.font.sizes.normal};
    margin-top: ${theme.spacings.small};
  `}
`;
