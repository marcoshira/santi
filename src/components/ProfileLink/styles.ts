import styled, { css } from 'styled-components';
import { Title } from '../Heading/styles';
import { Wrapper as Btn } from '../Button/styles';

export const Wrapper = styled.form`
  ${({ theme }) => css`
    box-shadow: 3px 3px 15px ${theme.colors.teal};
    border-radius: 15px;
    background: ${theme.colors.footerWhite};
    width: 45%;
    padding: ${theme.spacings.ltlarge};
    padding-top: ${theme.spacings.small};
    position: relative;
    padding-bottom: ${theme.spacings.xhero};

    margin-left: 50rem;
    margin-top: 15rem;

    @media ${theme.media.lteMedium} {
      margin: 13rem auto;
      width: 85%;
      height: 45rem;
    }

    p {
      position: absolute;
      bottom: 0.5rem;
      left: 2rem;

      @media ${theme.media.lteMedium} {
        bottom: 6rem;
        left: 3rem;
      }
    }

    ${Title} {
      margin-top: 0;
    }

    ${Btn} {
      a {
        text-decoration: none;
        color: ${theme.colors.footerWhite};
      }
      position: absolute;
      bottom: 1.5rem;
      right: 4rem;
      /* width: 15rem; */
      border: 2px solid ${theme.colors.teal};
      background: ${theme.colors.teal};
      color: ${theme.colors.footerWhite};
      border-radius: ${theme.spacings.tiny};
      margin-left: 0;
      &:hover {
        background: ${theme.colors.hoverTeal};
      }
      &::after {
        height: 0;
      }
      &:disabled {
        background: ${theme.colors.mediumGrey};
        color: ${theme.colors.footerWhite};
        border: 2px solid ${theme.colors.mediumGrey};

        &:hover {
          background: ${theme.colors.mediumGrey};
          color: ${theme.colors.white};
          border: 2px solid ${theme.colors.mediumGrey};
          cursor: not-allowed;
        }

        &::after {
          content: none;
        }
      }

      @media ${theme.media.lteMedium} {
        left: 3rem;
        right: 3rem;
      }
    }
  `}
`;
