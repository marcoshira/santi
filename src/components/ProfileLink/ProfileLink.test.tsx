import { fireEvent, screen, waitFor } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import { ProfileLink } from '.';
import { ProfileComponentProps } from '../../shared-types/shared-types';
import mock from '../ProfileComponent/mock';

const func = jest.fn();

const props: ProfileComponentProps = mock.usersPermissionsUser.data.attributes;

describe('<ProfileLink />', () => {
  it('should render', () => {
    renderTheme(<ProfileLink {...props} />);

    expect(
      screen.getByRole('heading', { name: 'My Profile' }),
    ).toBeInTheDocument();
  });
  it('should call function', async () => {
    renderTheme(<ProfileLink onSubmit={func} {...props} />);
    fireEvent.click(screen.getByRole('button'));

    await waitFor(() => {
      expect(func).toHaveBeenCalled();
    });
  });
  it('should change input', async () => {
    renderTheme(<ProfileLink onSubmit={func} {...props} />);
    const input = screen.getByLabelText('Username');
    const input2 = screen.getByLabelText('Telephone');
    const input3 = screen.getByLabelText('Address');
    fireEvent.change(input, { target: { value: 'abc' } });
    fireEvent.change(input2, { target: { value: 'abd' } });
    fireEvent.change(input3, { target: { value: 'abe' } });

    expect(input).toHaveValue('abc');
    expect(input2).toHaveValue('abd');
    expect(input3).toHaveValue('abe');
  });
  it('should render with errorMessage', () => {
    const { container } = renderTheme(
      <ProfileLink errorMessage="oi" {...props} />,
    );
    expect(screen.getByText('oi')).toBeInTheDocument();

    expect(container).toMatchSnapshot();
  });
});
