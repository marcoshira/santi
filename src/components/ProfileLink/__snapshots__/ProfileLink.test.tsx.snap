// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileLink /> should render with errorMessage 1`] = `
.c10 {
  padding: 0.8rem 1rem;
  background: inherit;
  color: #8c4bff;
  border: none;
  font-size: 2rem;
  font-weight: 500;
  margin-left: 0.8rem;
  margin-top: 5px;
  margin-bottom: 4px;
  position: relative;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c10:disabled {
  background: #DDDDDD;
  color: #FFFFFF;
  border: 2px solid #DDDDDD;
  -webkit-transition: none;
  transition: none;
}

.c10:disabled:hover {
  background: #DDDDDD;
  color: #FFFFFF;
  border: 2px solid #DDDDDD;
  cursor: not-allowed;
}

.c10:disabled::after {
  content: none;
}

.c10:hover {
  color: #ff4a4a;
  cursor: pointer;
}

.c10::after {
  content: '';
  position: absolute;
  bottom: 0.2rem;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #ff4a4a;
  -webkit-transition: all 200ms ease-in-out;
  transition: all 200ms ease-in-out;
}

.c10:hover::after {
  left: 25%;
  width: 50%;
}

.c2 {
  color: inherit;
  font-size: 3.2rem;
  text-transform: none;
}

.c3 {
  width: 100%;
  margin-top: 3.2rem;
}

.c4 {
  position: relative;
}

.c4 > svg {
  position: absolute;
  top: 50%;
  -webkit-transform: translate(0,-50%);
  -ms-transform: translate(0,-50%);
  transform: translate(0,-50%);
  right: 1rem;
  width: 2.5rem;
  height: 2.5rem;
  color: rgb(141,141,144);
  z-index: 1;
  color: rgb(247,38,115);
}

.c4 *:focus ~ svg {
  color: #8c4bff;
  color: rgb(247,38,115);
}

.c4 *:disabled ~ svg {
  color: rgb(201,201,203);
}

.c5 {
  border: 1px solid rgb(201,201,203);
  width: 100%;
  height: 100%;
  font-size: 1.8rem;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
  padding: 1.6rem 1rem;
  background: #FFFFFF;
  padding-right: 3.7rem;
  border-radius: 0.4rem;
  outline: none;
  border: 0.1rem solid rgb(247,38,115);
  box-shadow: 0 0 0.2rem 0 rgb(247,38,115);
}

.c5::-webkit-input-placeholder {
  visibility: hidden;
  opacity: 0;
}

.c5::-moz-placeholder {
  visibility: hidden;
  opacity: 0;
}

.c5:-ms-input-placeholder {
  visibility: hidden;
  opacity: 0;
}

.c5::placeholder {
  visibility: hidden;
  opacity: 0;
}

.c5:focus {
  border: 0.1rem solid #8c4bff;
  box-shadow: 0 0 0.2rem 0 #8c4bff;
  background: #FFFFFF;
}

.c5:focus + .c6,
.c5:not(:placeholder-shown) + .c6,
.c5:-webkit-autofill + .c6 {
  top: 0;
  -webkit-transform: translate(0,-50%);
  -ms-transform: translate(0,-50%);
  transform: translate(0,-50%);
  font-size: 1.2rem;
  color: #FFFFFF;
  background: #8c4bff;
}

.c5:disabled {
  background: rgb(241,241,243);
  color: rgb(161,161,164);
}

.c5:disabled + .c6 {
  background: rgb(161,161,164);
  color: rgb(221,221,223);
}

.c5:focus {
  border: 0.1rem solid rgb(247,38,115);
  box-shadow: 0 0 0.2rem 0 rgb(247,38,115);
}

.c5:focus + .c6,
.c5:not(:placeholder-shown) + .c6 {
  color: #FFFFFF;
  background: rgb(247,38,115);
}

.c7 {
  display: inline;
  position: absolute;
  top: 50%;
  -webkit-transform: translate(0,-50%);
  -ms-transform: translate(0,-50%);
  transform: translate(0,-50%);
  left: 1rem;
  font-size: 1.6rem;
  height: 1.6rem;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
  background: #FFFFFF;
  padding: 0.5rem 1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  line-height: 1;
  z-index: 1;
  color: rgb(141,141,144);
  border-radius: 0.4rem;
}

.c8 {
  color: rgb(247,38,115);
  margin: 0.8rem 0 3.2rem;
}

.c0 {
  box-shadow: 3px 3px 15px #8c4bff;
  border-radius: 15px;
  background: #FFFFFF;
  width: 45%;
  padding: 2.8rem;
  padding-top: 1.6rem;
  position: relative;
  padding-bottom: 8.0rem;
  margin-left: 50rem;
  margin-top: 15rem;
}

.c0 p {
  position: absolute;
  bottom: 0.5rem;
  left: 2rem;
}

.c0 .c1 {
  margin-top: 0;
}

.c0 .c9 {
  position: absolute;
  bottom: 1.5rem;
  right: 4rem;
  border: 2px solid #8c4bff;
  background: #8c4bff;
  color: #FFFFFF;
  border-radius: 0.4rem;
  margin-left: 0;
}

.c0 .c9 a {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #FFFFFF;
}

.c0 .c9:hover {
  background: #6d39c7;
}

.c0 .c9::after {
  height: 0;
}

.c0 .c9:disabled {
  background: #DDDDDD;
  color: #FFFFFF;
  border: 2px solid #DDDDDD;
}

.c0 .c9:disabled:hover {
  background: #DDDDDD;
  color: #FFFFFF;
  border: 2px solid #DDDDDD;
  cursor: not-allowed;
}

.c0 .c9:disabled::after {
  content: none;
}

@media (max-width:768px) {
  .c0 {
    margin: 13rem auto;
    width: 85%;
    height: 45rem;
  }
}

@media (max-width:768px) {
  .c0 p {
    bottom: 6rem;
    left: 3rem;
  }
}

@media (max-width:768px) {
  .c0 .c9 {
    left: 3rem;
    right: 3rem;
  }
}

<div>
  <form
    class="c0"
  >
    <h1
      class="c1 c2"
    >
      My Profile
    </h1>
    <div
      class="c3"
    >
      <div
        class="c4"
      >
        <input
          class="c5"
          id="user-username"
          name="user-username"
          placeholder="Username"
          type="text"
          value="luisa"
        />
        <label
          class="c6 c7"
          for="user-username"
        >
          Username
        </label>
      </div>
      <p
        class="c8"
      >
         
      </p>
    </div>
    <div
      class="c3"
    >
      <div
        class="c4"
      >
        <input
          class="c5"
          id="user-phone"
          name="user-phone"
          placeholder="Telephone"
          type="tel"
          value="2222 2222"
        />
        <label
          class="c6 c7"
          for="user-phone"
        >
          Telephone
        </label>
      </div>
      <p
        class="c8"
      >
         
      </p>
    </div>
    <div
      class="c3"
    >
      <div
        class="c4"
      >
        <input
          class="c5"
          id="user-address"
          name="user-address"
          placeholder="Address"
          type="text"
          value="Rua BarÃ£o de spdaksadopk"
        />
        <label
          class="c6 c7"
          for="user-address"
        >
          Address
        </label>
      </div>
      <p
        class="c8"
      >
        oi
      </p>
    </div>
    <button
      class="c9 c10"
    >
      Save Changes
    </button>
    <p>
      Created at: 
      2022-05-16
    </p>
  </form>
</div>
`;
